spec/models/topic_view_item_spec.rb

```diff
@@ -6,6 +6,33 @@ def add(topic_id, ip, user_id = nil)
     TopicViewItem.add(topic_id, ip, user_id, nil, skip_redis)
   end
 
+  it "correctly increase topic view stats" do
```

Comment:
I think it's worth adding some request specs here too:

https://github.com/discourse/discourse/blob/dd2fa58dd187a1298c56536a98dbd13ca11c4012/spec/requests/topics_controller_spec.rb#L2700-L2702

Just to ensure that both anon and logged in user views are logged correctly from that level

Reply:
this is going to break when we amend "how we count", we will be forced to fix the test to add a header, might as well just do the testing then.